app/models/channel.js

```diff
@@ -8,6 +8,9 @@ export default Ember.Object.extend({
   connected: false,
   sockethubChannelId: null,
   topic: null,
+  unreadMessages: false,
+  unreadMentions: false,
+  visible: false, // Current/active channel
```

Comment:
Not sure `visible` is a good choice. Shouldn't it rather be `active`?

`visible` makes me think you could also hide channels somehow.